docs/pages/api.rst

```diff
@@ -1268,7 +1268,7 @@ Signature: ``Collection::intersperse($element, int $every = 1, int $startAt = 0)
 isEmpty
 ~~~~~~~
 
-Check if a collection has any elements inside.
+Check if a collection has no elements inside.
```

Comment:
I like this distinction, but please update the one on the `IsEmptyable` interface as well :)